src/frontend/src/routes/(app)/+page.svelte

```diff
@@ -1,5 +1,10 @@
 <script lang="ts">
+	import DappsCarousel from '$lib/components/dapps/DappsCarousel.svelte';
 	import Tokens from '$lib/components/tokens/Tokens.svelte';
 </script>
 
+<div class="mb-6 flex justify-center xl:hidden">
```

Comment:
Same question, does it requires a fade effect?

Reply:
Yep, added.